app/index.js

```diff
@@ -1447,6 +1454,18 @@ module.exports = JhipsterGenerator.extend({
             this.removefile('profile_prod.gradle');
             this.removefile('sonar.gradle');
             this.removefile('yeoman.gradle');
+        },
+
+        writeEntitiesFiles: function () {
```

Comment:
@cbornet  you need do this in the end method to avoid run loop issues, so that entity regen will happen only after main generator has run fully